### ISSUE TITLE
Unable to compute current data using extract_tidal_constants

### ISSUE THREAD

**Issue by mmr0** [tsutterley/pyTMD#114]
Hi,

Really appreciate your work with this as the Matlab TMD package has been the only reason I've kept Matlab installed for the past 3 years. 

I have no problems getting elevations using this function, but I don't get anything meaningful when I try to get currents, e.g.

```
lat = [-66.2]
lon = [71.45]

model_grid = '/Users/.../py_nb/TIDES/CATS2008/grid_CATS2008'
model_elevation = '/Users/.../py_nb/TIDES/CATS2008/hf.CATS2008.out'
model_current = '/Users/.../py_nb/TIDES/CATS2008/uv.CATS2008.out'

amp,ph,D,c = extract_tidal_constants(lon, lat, model_grid, model_elevation, 'CATS2008', type='z',
            method='spline', grid='OTIS', extrapolate=True)
print('elevation: ', np.round(D), amp[:,0], amp[:,4])

amp,ph,D,c = extract_tidal_constants(lon, lat, model_grid, model_current, 'CATS2008', type='u', 
           method='spline', grid='OTIS')
print('current (u):', np.round(D), amp[:,0], amp[:,4])
```
which outputs:

```
elevation:  [2320.] [0.1318957358598709] [0.22610248625278473]
current (u): [2320.] [--] [--]
```

Not sure if this is a bug or a mistake on my part? But I seem to have the same issue with the "Plot Antarctic Tidal Currents" notebook. 

Cheers

**Comment by tsutterley** [comment]
Hey @mmr0,
Yep I broke the interpolation for OTIS currents in [d5d3390](https://github.com/tsutterley/pyTMD/commit/d5d33902d19d1185fc34f3c6dca1d140aab1ab19) when I was trying to get better estimates near the coasts. This is (hopefully) fixed in #115. Sorry for the trouble and I'm glad you're finding pyTMD useful.
-ts

**Comment by mmr0** [comment]
Thanks, working now!
